### DISCUSSION TITLE
What about server-side rendering?

### DISCUSSION THREAD

**Discussion by realconvis** [gtm-support/vue-gtm#377] [Q&A]
I wonder what about server-side rendering.

Does the page view events via router fire on server side rendering?

Did i get it right, if i use gtm.push() for my own custom events, they should be placed in onMounted() to avoid that the push events fires on server side and client side?
In my opinion it only make sence that gtm events fires only on client side.

**Comment by Shinigami92** [comment]
I don't know how to distinguish between ssr and client, but I suggest you set `enabled` based on if it is client

**Reply by realconvis** [reply]
Oh, sorry.
I forgot to mention that I'm using the Nuxt module
Nuxt is responsible for server side rendering. 
Maybe I should ask the question on the Nuxt Module Github..
